<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditor</class>
 <widget class="QDialog" name="TagEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Dialog title">Edit tags</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="trackPerformerLabel">
       <property name="text">
        <string>Artist:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="TrackTagLineEdit" name="performerEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="diskAPerformerLabel">
       <property name="text">
        <string comment="Music tag name" extracomment="Music tag name">Album performer:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="AlbumTagLineEdit" name="albumPerformerEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="trackSongWriterLabel">
       <property name="text">
        <string>Song writer:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="TrackTagLineEdit" name="songWriterEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="diskAlbumLabel">
       <property name="text">
        <string comment="Music tag name" extracomment="Music tag name">Album:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="AlbumTagLineEdit" name="albumEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="trackDateLabel">
       <property name="text">
        <string>Year:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="TrackTagLineEdit" name="dateEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="trackGnreLabel">
       <property name="text">
        <string>Genre:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="TrackTagLineEdit" name="genreEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="trackTitleLabel">
       <property name="text">
        <string>Track title:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="TrackTagLineEdit" name="trackTitleEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="diskTracksNumsLabel">
       <property name="text">
        <string comment="Music tag name" extracomment="Music tag name">Start track number:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="MultiValuesSpinBox" name="startTrackEdit">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>of</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MultiValuesSpinBox" name="trackCountEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="diskNumsLabel">
       <property name="text">
        <string comment="Music tag name" extracomment="Music tag name">Disc number:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="MultiValuesSpinBox" name="diskNumEdit">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>of</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MultiValuesSpinBox" name="diskCountEdit">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="trackCommentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="TrackTagTextEdit" name="commentEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiValuesSpinBox</class>
   <extends>QSpinBox</extends>
   <header>controls.h</header>
  </customwidget>
  <customwidget>
   <class>TrackTagLineEdit</class>
   <extends>QLineEdit</extends>
   <header>controls.h</header>
  </customwidget>
  <customwidget>
   <class>TrackTagTextEdit</class>
   <extends>QTextEdit</extends>
   <header>controls.h</header>
  </customwidget>
  <customwidget>
   <class>AlbumTagLineEdit</class>
   <extends>QLineEdit</extends>
   <header>controls.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TagEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TagEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
